sources/packages/backend/apps/api/src/services/student-file/student-file.service.ts

```diff
@@ -152,6 +152,19 @@ export class StudentFileService extends RecordDataModelService<StudentFile> {
     return this.repo.findOne({ where: { uniqueFileName: uniqueFileName } });
   }
 
+  /**
```

Comment:
We can re-use this existing method (just add friendly name to select.) 

![image](https://github.com/user-attachments/assets/ecd8639d-f21c-404e-a0b1-72791b17a4c7)